<launch>
<param name="multi_agent_model" type="string" value="pure"/>
<param name="number_of_mines" type="int" value="10"/>
<param name="event_time_horizon" type="int" value="30"/>
<param name="number_of_charging_docks" type="int" value="2"/>
<param name="agent_policy_steps" type="double" value="200"/>
<param name="agent_poll_time" type="double" value="30"/>
<param name="agent_step_time" type="double" value="1"/>

<param name="agent_interaction_length" type="double" value="1"/>

<param name="agent_trajectory_length" type="double" value="4"/>
<param name="total_trial_time" type="double" value="120"/>
<param name="load_q" type="bool" value="False"/>
<param name="write_q" type="bool" value="True"/>
<param name="write_psi" type="bool" value="True"/>
<param name="temp_file_path" type="String" value="/home/aaron/mines_temp"/>


<node pkg="mines" type="main.py" name="main" output="screen">
</node>

<node pkg="mines" type="agent_standard.py" name="a1" output="screen">
</node>






</launch>
